CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # IdentityCache changelog
 
+#### 0.4.0
+
+- Return an array from fetched association to prevent chaining. Up to now, a relation was returned by default. (#287)
+
 #### 0.3.2
```

Comment:
Already pushed the v.0.3.2 tag. Shipit fails to pick up the tag. We might need to use the traditonal `rake release` while the issue is being debugged.